lib/mail/BatchId.php

```diff
@@ -48,7 +48,7 @@ public function __construct($batch_id = null)
     public function setBatchId($batch_id)
     {
         if (!is_string($batch_id)) {
-            throw new TypeException('$batch_id must be of type string.');
+            throw new TypeException('$batch_id must be a string.');
```

Comment:
I don't this exception messages are necessary to be changed.

If possible, can you fix all these related messages? I think some messages have the same problems, too :).